### PR TITLE
Views

### PR DESCRIPTION
**Pull request by imubarak234**
Implement the design from the sneak peek wireframes.

Styling is nice to have but focus on making sure everything functions first.

Use methods that you have created in the Blog App - processing data in models 
Hint: In order to see some posts or comments, you might need to add some data to your database as described

### CODE REVIEW

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: INVALID  ❌ 


###  @imubarak234  ✋🏼 
#### Your project is not working. I add a sample in the seed file. After ı run `rails db:seed` I am getting a syntax error from the post model. Probably you fix rubocop automatically and it deletes some codes. Please fix the errors and test your project and send a review again. 
Example:
image
 
#### Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏


### Cheers and Happy coding! 💻 👏

coding

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

_Invalid Code Review Request **does not count** into the code reviews limit._

___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Comment by imubarak234** [comment]
We haven't been taught anything about seed files. Since we started this Module two weeks and ago there has been no mention of seed file. We were taught to pupolate our tables [manually](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,9 +1,22 @@
 class PostsController < ApplicationController
   def index
-    render 'index'
+    @posts = Post.new.print
+    @users = User.new.info(params[:user_id])
+    # @post_comments
```
Comment (andersonlebon) [inline app/controllers/posts_controller.rb]: - [ ] KIndly remove commented code to clear you code.

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -1,9 +1,22 @@
 class PostsController < ApplicationController
   def index
-    render 'index'
+    @posts = Post.new.print
+    @users = User.new.info(params[:user_id])
```
Comment (andersonlebon) [inline app/controllers/posts_controller.rb]: - [ ] Kindly in Ruby on rails the syntax for getting info in a database table we use all methods on the modal. like `User.all`. and to find a specific user we use user.fin(user_id) . in your case you should use :point_down: 

```rb
 def index
    @posts = Post.all
    @user = User.find(params[:user_id])
end
```
- [ ] After this action :point_up: ,make sure the correspondent view is working .

**Comment by ahmetbozaci** [comment]
> We haven't been taught anything about seed files. Since we started this Module two weeks and ago there has been no mention of seed file. We were taught to pupolate our tables [manually](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data).

- The seed file is in the `db` folder. You can just write samples in this file and  run `rails db:seed` 
- Instead of creating one by one in the rails console, you can create like that. 

**The point is your model files have some syntax errors so you cant create users or posts. You can test in the rails console.**
image

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -1,9 +1,22 @@
...
+    Post.new.recent_comments_all(id)
+  end
+
+  helper_method :comments, :comments_all
+
   def show
-    render 'show'
+    @post_show = Post.new.info(params[:id])
+    @user_show = User.new.info(params[:user_id])
   end
```
Comment (andersonlebon) [inline app/controllers/posts_controller.rb]: - [ ] Kindly in Ruby on rails the syntax for getting info in a database table we use all methods on the modal. like `User.all`. and to find a specific user we use user.find(user_id). in your case you should use :point_down: 

```rb
 def index
    @ = Post.all
    @user = User.find(params[:user_id])
end
```
- [ ] After this action :point_up: ,make sure the correspondent view is working .

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -1,9 +1,13 @@
 class UsersController < ApplicationController
   def index
-    render 'index'
+    # render 'index'
+    @user = User.new
+    @user = @user.print
```
Comment (andersonlebon) [inline app/controllers/users_controller.rb]: - [ ] Kindly in Ruby on rails the syntax for getting info in a database table we use all methods on the modal. like `User.all`. and to find a specific user we use user.find(user_id). in your case you should use :point_down: 
example:

```rb
 def index
    @ = Post.all
    @user = User.find(params[:user_id])
end
```
- [ ] After this action :point_up: ,make sure the correspondent view is working .

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @imubarak234 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (rloterh) [reply app/controllers/posts_controller.rb]: This issue is well resolved! ✔️

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (rloterh) [reply app/controllers/posts_controller.rb]: This issue is well resolved! ✔️

File: app/controllers/posts_controller.rb (hunk 3 above)
Comment (rloterh) [reply app/controllers/posts_controller.rb]: This issue is well resolved! ✔️

File: app/controllers/users_controller.rb (hunk 4 above)
Comment (rloterh) [reply app/controllers/users_controller.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Mubarak, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**